python/katana/_property_graph_numba.pyx.jinja

```diff
@@ -14,10 +15,10 @@ from libc.stdint cimport uint32_t, uint64_t
     return self.topology().num_edges()
 {% endcall %}
 {% call numba.method_with_body("edge_index", "uint64_t", ["uint64_t"]) %}
-    return self.topology().out_indices.get().Value(arg1)
+    return deref(self.topology().edge_end(arg1))
```

Comment:
This needs to be `self.topology().edges(arg1).end()` I didn't expose `edge_end` in python because I hope it's going away.